qualtran/cirq_interop/_cirq_to_bloq.py

```diff
@@ -378,6 +378,11 @@ def _cirq_gate_to_bloq(gate: cirq.Gate) -> Bloq:
     if gate in CIRQ_GATE_TO_BLOQ_MAP:
         return CIRQ_GATE_TO_BLOQ_MAP[gate]
 
+    if isinstance(gate, cirq.ControlledGate):
+        return Controlled(
+            cirq_gate_to_bloq(gate.sub_gate), CtrlSpec.from_cirq_cv(gate.control_values)
+        )
+
```

Comment:
Why this change?

Reply:
for #1115  or any case where we have an explicit bloq that takes priority over the general `Controlled` wrap.